### PR TITLE
SCA-2 Create project skeleton

### PR DESCRIPTION
**Pull request by KaterynaHolt**
The project skeleton has been created

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```
Comment (aholtvianskyi) [inline pom.xml:24]: Could you move this version to properties section?

File: src/main/java/org/example/App.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+
+public class App 
+{
```
Comment (aholtvianskyi) [inline src/main/java/org/example/App.java]: Please use open curly braces after the name of the class/method. E.g.

public class App {
...
}

File: src/test/java/org/example/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
+{
```
Comment (aholtvianskyi) [inline src/test/java/org/example/AppTest.java]: The same about curly braces as I've mentioned above.

File: src/test/java/org/example/AppTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (aholtvianskyi) [inline src/test/java/org/example/AppTest.java:10]: It's good that you've added the unit test to your class but what is the purpose of it?

File: pom.xml (hunk 1 above)
Comment (KaterynaHolt) [reply pom.xml:24]: Ok, I've changed it

File: src/main/java/org/example/App.java (hunk 2 above)
Comment (KaterynaHolt) [reply src/main/java/org/example/App.java]: Ok, I've changed it

File: src/test/java/org/example/AppTest.java (hunk 4 above)
Comment (KaterynaHolt) [reply src/test/java/org/example/AppTest.java:10]: This file was generated automatically by Maven

**Comment by aholtvianskyi** [comment]
Approved

**Pull request merged**